docs/docsite/rst/scenario_guides/guide_infoblox.rst

```diff
@@ -0,0 +1,304 @@
+            ipv4addrs:
+              - ipv4addr:
+                  "{{ lookup('nios_next_ip', '192.168.100.0/24', provider=nios_provider)[0] }}"
+            state: present
+    provider: "{{nios_provider}}"
+
+Notice the IPv4 address in this example uses the ``nios_next_ip`` lookup plugin to find the next available IPv4 address on the network. You can find complete details on the ``nios_next_ip`` lookup plugin at :ref:`nios_next_ip <nios_next_ip_lookup>`.
+
+
+Creating a forward DNS zone
+---------------------------
+
+The following example playbook uses the ``nios_zone`` module to configure a forward DNS zone:
+
+.. code-block:: yaml
+
+    ---
+    - hosts: nios
+      connection: local
+      tasks:
+        - name: Create a forward DNS zone called ansible-test.com
+          nios_zone:
+            name: ansible-test.com
+            comment: local DNS zone
+            state: present
+            provider: "{{ nios_provider }}"
+
+Creating a reverse DNS zone
+---------------------------
+
+The following example playbook uses the ``nios_zone`` module to configure a reverse DNS zone:
+
+.. code-block:: yaml
+
+    ---
+    - hosts: nios
+      connection: local
+      tasks:
+        - name: configure a reverse mapping zone on the system using IPV6 zone format
+          nios_zone:
+            name: 100::1/128
+            zone_format: IPV6
+            state: present
+            provider: "{{ nios_provider }}"
+
+Dynamic inventory script
+========================
+
+You can use the Infoblox dynamic inventory script to import your network node inventory with Infoblox NIOS. To gather the inventory from Infoblox, you need two files:
+
+- `infoblox.yaml <https://raw.githubusercontent.com/ansible/ansible/devel/contrib/inventory/infoblox.yaml>`_ - A file that specifies the NIOS provider arguments and optional filters.
+
+- `infoblox.py <https://raw.githubusercontent.com/ansible/ansible/devel/contrib/inventory/infoblox.py>`_ - The python script that retrieves the NIOS inventory.
+
+To use the Infoblox dynamic inventory script:
+
+1. Download the ``infoblox.yaml`` file and save it in the ``/etc/ansible`` directory.
+
+2. Modify the ``infoblox.yaml`` file with your NIOS credentials.
+
+3. Download the ``infoblox.py`` file and save it in the ``/etc/ansible/hosts`` directory.
+
+4. Change the permissions on the ``infoblox.py`` file to make the file an executable:
+
+.. code-block:: bash
+
+    $ sudo chmod +x /etc/ansible/hosts/infoblox.py
+
+5. Optionally, test the script:
+
+.. code-block:: bash
+
+   $  ./infoblox.py --list
+
+After a few minutes, you should see your Infoblox inventory in JSON format.
+
+You can explicitly use the Infoblox dynamic inventory script as follows:
+
+.. code-block:: bash
+
+    $ ansible -i infoblox.py all -m ping
+
+You can also implicitly use the Infoblox dynamic inventory script by including it in your inventory directory (``etc/ansible/hosts`` by default).
+
+See :ref:`dynamic_inventory` for more details.
```

Comment:
I feel like the whitespace in the rendered HTML makes it hard to parse the page - maybe try having fewer paragraphs/newlines?